code/game/objects/effects/temporary_visuals/miscellaneous.dm

```diff
@@ -489,7 +489,7 @@
 	duration = 2 SECONDS
 
 /obj/effect/constructing_effect
-	icon = 'icons/effects/rcd.dmi'
+	icon = 'icons/effects/rcd_tall.dmi'
```

Comment:
this is the only place that uses this file, you should just override the old dmi no? or like